api/report_server.thrift

```diff
@@ -170,18 +177,18 @@ service codeCheckerDBAccess {
                        1: i64 reportId)
                        throws (1: shared.RequestFailed requestError),
 
-  // get the results for one runId
+  // get the results for some runIds
   ReportDataList getRunResults(
-                               1: i64 runId,
+                               1: list<i64> runIds,
```

Comment:
Agree. In this context, `optional` doesn't add extra information. It's also not harder for the protocol to send either, so...